### ISSUE TITLE
Change License to MIT

### ISSUE THREAD

**Issue by thomthom** [SketchUp/sketchup-ply#8]
@stoutb77 : We need to reach out to Jim to change this license to MIT as well before we can go public with it. It was a plugin from Jim, right?

**Comment by jimfoltz** [comment]
> It was a plugin from Jim, right?

Yes it was.  I agree to the proposed change in license to the MIT open-source license for this project.

**Comment by stoutb77** [comment]
I will blast out an email to all contributors tomorrow.
